<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Mp_Hochladen</class>
 <widget class="QDialog" name="Mp_Hochladen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>504</width>
    <height>331</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Set hochladen</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Set hochladen</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="lbl_name">
       <property name="text">
        <string>Name:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txt_name"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="lbl_sprache">
       <property name="text">
        <string>Sprache:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cmb_sprache">
       <item>
        <property name="text">
         <string>Englisch</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Französisch</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Latein</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Spanisch</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Italienisch</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="lbl_tags">
     <property name="text">
      <string>Beschreibung / Tags:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="txt_tags"/>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <italic>true</italic>
      </font>
     </property>
     <property name="text">
      <string>Schreiben Sie hier Wörter rein, die zu dem Set passen und vereinfachen Sie damit die Suche danach.</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="cmd_hochladen">
       <property name="text">
        <string>Hochladen</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cmd_abbrechen">
       <property name="text">
        <string>Abbrechen</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
